### PR TITLE
add test run autoсlosing

### PR DESCRIPTION
**Pull request by Igur007**

### CODE REVIEW

**Comment by Igur007** [comment]
@dubner is there a chance to have this merged?

**Comment by dubner** [comment]
@Igur007 
Would prefer this as a flag, default I suspect is best set to True.

**Comment by Igur007** [comment]
@dubner review, please. Thanks in advance!

File: pytest_testrail/plugin.py [hunk 1]
```diff
@@ -181,6 +183,10 @@ def pytest_sessionfinish(self, session, exitstatus):
                     self.add_results(testrun_id)
             else:
                 print('[{}] No data published'.format(TESTRAIL_PREFIX))
+            
+            if self.close_on_complete:
```
Comment (apallier) [inline pytest_testrail/plugin.py]: Why only the last testrun is closed? What behavior is excepted when '`--tr-plan-id`' option is used?

File: tests/test_plugin.py [hunk 2]
```diff
@@ -260,3 +260,13 @@ def test_get_available_testruns(api_client, tr_plugin):
     testplan_id = 100
     api_client.send_get.return_value = TESTPLAN
     assert tr_plugin.get_available_testruns(testplan_id) == [59, 61]
+
+
+def test_close_test_run(api_client, tr_plugin):
+    api_client.send_get.return_value = {'is_completed': False}
+    assert tr_plugin.is_testrun_available() is True
+    
+    tr_plugin.close_test_run()
```
Comment (apallier) [inline tests/test_plugin.py]: Sorry but here you are not testing `close_test_run` but `is_testrun_available` :)
We should instead test the POST query triggered by `close_test_run`.
Look examples at [line 174](https://github.com/Igur007/pytest-testrail/blob/94bdac8cb8ae2c7199464f6bfe39089886a7d6be/tests/test_plugin.py#L174) or [line 202](https://github.com/Igur007/pytest-testrail/blob/94bdac8cb8ae2c7199464f6bfe39089886a7d6be/tests/test_plugin.py#L202)

**Comment by Igur007** [comment]
@apallier thanks for feedback, hope that fixed). New review is needed.

File: pytest_testrail/conftest.py [hunk 3]
```diff
@@ -97,7 +100,8 @@ def pytest_configure(config):
                 tr_name=config_manager.getoption('tr-testrun-name', 'name', 'TESTRUN'),
                 run_id=config.getoption('--tr-run-id'),
                 plan_id=config.getoption('--tr-plan-id'),
-                version=config.getoption('--tr-version')
+                version=config.getoption('--tr-version'),
+                close_on_complete=config.getoption('--close-on-complete')
```
Comment (dubner) [inline pytest_testrail/conftest.py]: command line options, should be prefixed with 'tr'

Comment (Igur007) [reply pytest_testrail/conftest.py]: @dubner  fixed, thanks

Comment (Igur007) [reply pytest_testrail/conftest.py]: @dubner please review changes.

**Comment by Igur007** [comment]
@dubner Does some problem block this request or is there any other unfixed issue?

**Comment by apallier** [comment]
@Igur007 Please, could you fix the bug with `testrun_name` in `plugin.py` [line 295](https://github.com/Igur007/pytest-testrail/blob/5d81b381570efb3a14ffc097511add8074048d5a/pytest_testrail/plugin.py#L295)? 
After that it will be OK from my side.

**Comment by Igur007** [comment]
@apallier @dubner fixed the bug with testrun_name in plugin.py line 295

**Comment by apallier** [comment]
@Igur007 I'm so sorry but your correction doesn't work with options `--tr-run-id` or `--tr-plan-id` because in this case `self.testrun_name` is `None`.
I propose to you a simpler solution that works in all cases:
``` python
print('[{}] Test run with ID={} was closed'.format(TESTRAIL_PREFIX, self.testrun_id))
```
Thank you.

**Comment by Igur007** [comment]
@apallier got it. Thanks, forgot that If --tr-run-id or --tr-plan-id provided, option "--tr-testrun-name" will be ignored.

**Comment by apallier** [comment]
@allankp Don't you want to merge this PR?

**Pull request merged**